k8s-pb-codegen/src/main.rs

```diff
@@ -95,5 +107,172 @@ fn main() -> Result<()> {
         std::fs::rename(src, dst)?;
     }
 
+    let mut lib_rs = OpenOptions::new()
+        .append(true)
+        .open(target_dir.join("lib.rs"))
+        .unwrap();
+    append_trait_def(&mut lib_rs);
+
     Ok(())
 }
+
+fn append_trait_def(lib_rs: &mut File) {
```

Comment:
hah, that is a nice way to get started. We can think about where to move this stuff later in https://github.com/kube-rs/k8s-pb/issues/4